README.md

```diff
@@ -34,6 +34,26 @@ app.use('/pdf', (req, res) => {
 })
 ```
 
+### Configuring Puppeteer
+Puppeteer launch options (including Chromium flags via the args array) can be passed to the pdfRenderer function and used on every render.
+If no options are specified the default options below will be used.
+
+**Note: The default puppeteer launch options pass flags to disable the Chrome sandbox**
+
+This is one way to enable running Puppeteer in Docker but may be a security issue if you are loading untrusted content, in which case you should override these defaults.
+See [Puppeteer troubleshooting](https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#setting-up-chrome-linux-sandbox) for further info on the use of --no-sandbox
+
+Default options:
+```javascript
+{ args: ['--no-sandbox', '--disable-setuid-sandbox'] }
+``` 
+
+Pass options to the pdfRendered function to replace the defaults. Pass an empty object if you only want to remove the defaults.
+See [Puppeteer launch options](https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/docs/api.md#puppeteerlaunchoptions) for more info
+```javascript
+app.use(pdfRenderer( { timeout:60000 } ))
```

Comment:
Can we also default the timeout?

Reply:
There already is a default of 30000 (30 seconds). This was just meant to be an example of how you could set other options

Reply:
Clarified the readme